components/policy/brave_browser_policy_provider.cc

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/policy/brave_browser_policy_provider.h"
+
+#include <utility>
+
+#include "base/logging.h"
+#include "base/values.h"
+#include "components/policy/core/common/policy_bundle.h"
+#include "components/policy/core/common/policy_map.h"
+#include "components/policy/core/common/policy_namespace.h"
+#include "components/policy/core/common/policy_types.h"
+#include "components/prefs/pref_service.h"
+
+namespace brave_policy {
+
+BraveBrowserPolicyProvider::BraveBrowserPolicyProvider()
+    : policy::ConfigurationPolicyProvider(), first_policies_loaded_(false) {}
```

Comment:
nit: could use `bool first_policies_loaded_ = false;` in the header instead

Reply:
Fixed and pushed